Update several functions in io and analyzer

xingyu-liu opened the pull request.

xingyu-liu committed:
Merge pull request #3 from BNUCNL/framework

db_mv development

Commit:
update

Commit:
update

helloTC commented:
Please resolve conflicts and commit your code with understandable hints.

helloTC commented:
To any of your commits, please commit your modifications but rather use 'update', especially there are many modifications in your commits.

xingyu-liu committed:
Merge branch 'framework' into framework

Commit:
add required module

Commit:
fix some bugs

class PicDataset
1. change the input of as type dict
2. add conditional statement for key 'condition'
3. change the name of method read_Imagefolder to read_imagefolder

read_dnn_csv:
1. add conditional statement for array dtype of different type fo csvfile
2. change the error flag

Commit:
Merge branch 'framework' of https://github.com/xingyu-liu/dnnbrain into framework

Commit:
remove error flag for type response

Commit:
change blank into comma

Commit:
change blank into comma

Commit:
fix a typo

sunshineDrizzle commented:
1. 建议把.db.csv样板文件放到data目录里
2. dnn io文件第331行的操作应该是想让键名以小写字母开头，但是我觉得键值和dbcsv里的统一会更不容易犯错。
3. dnn io文件第341~346行可以更精简成四行代码

Commit:
lowercase the first letter of all keys

Commit:
adjust the layout to be compliant with PEP-8

Commit:
delete db.csv in bin/
